docker-compose.yml

```diff
@@ -380,46 +451,110 @@ services:
       ports:
         - "1025:1025"
         - "8025:8025"
+      networks:
+        - frontend
 
-### Selenium Container #########################################
+### Selenium Container ########################################
 
     selenium:
       build: ./selenium
       ports:
         - "${SELENIUM_PORT}:4444"
       volumes:
         - /dev/shm:/dev/shm
+      networks:
+        - frontend
+
+### Varnish Proxy 1 ##########################################
+
+    proxy:
+      build: ./varnish
+      expose:
+        - ${VARNISH_PORT}
+      environment:
+        - VARNISH_CONFIG=${VARNISH_CONFIG}
+        - CACHE_SIZE=${VARNISH_PROXY1_CACHE_SIZE}
+        - VARNISHD_PARAMS=${VARNISHD_PARAMS}
+        - VARNISH_PORT=${VARNISH_PORT}
+        - BACKEND_HOST=${VARNISH_PROXY1_BACKEND_HOST}
+        - BACKEND_PORT=${VARNISH_BACKEND_PORT}
+        - VARNISH_SERVER=${VARNISH_PROXY1_SERVER}
+      links:
+        - workspace
+      networks:
+        - frontend
+
+### Varnish Proxy 2 ##########################################
 
-### Volumes Setup ###########################################
+    proxy2:
+      build: ./varnish
+      expose:
+        - ${VARNISH_PORT}
+      environment:
+        - VARNISH_CONFIG=${VARNISH_CONFIG}
+        - CACHE_SIZE=${VARNISH_PROXY2_CACHE_SIZE}
+        - VARNISHD_PARAMS=${VARNISHD_PARAMS}
+        - VARNISH_PORT=${VARNISH_PORT}
+        - BACKEND_HOST=${VARNISH_PROXY2_BACKEND_HOST}
+        - BACKEND_PORT=${VARNISH_BACKEND_PORT}
+        - VARNISH_SERVER=${VARNISH_PROXY2_SERVER}
+      links:
+        - workspace
+      networks:
+        - frontend
+
```

Comment:
Wouldn't it be better to utilize the scalining features of docker-compose rather than repeating yourself? DRY and all that.